sampleconfig/config/config.yml

```diff
@@ -28,7 +28,7 @@ server:
     name: leveldb
     # database.ledgerDirectory denotes the root path
     # where we store all ledger data
-    ledgerDirectory: ledger
+    ledgerdirectory: /etc/bcdb-server/database
```

Comment:
let's use `ledger` instead of `database`

Reply:
ok